rasa/shared/core/training_data/structures.py

```diff
@@ -235,6 +236,11 @@ def is_action_unlikely_intent(event: Event) -> bool:
         return (
             type(event) == ActionExecuted
             and event.action_name == ACTION_UNLIKELY_INTENT_NAME
+        ) or (
+            type(event) == WronglyPredictedAction
```

Comment:
Do we still need this now we have `WarningPredictedAction`?

Reply:
Also do we need any check for these test events here? They are not actually predicted ever right?